### ISSUE TITLE
manifest.json isn't updated for subsequent rebuilds

### ISSUE THREAD

**Issue by gauravtiwari** [webdeveric/webpack-assets-manifest#28]
## The problem

manifest.json isn't updated for subsequent rebuilds when using watcher and dev server

## Technical details

In order to reproduce, 

1. Update to latest version
2. Run dev server or watcher for initial compilation
3. Change a word in the entry file, the manifest is updated fine
4. Revert change, the manifest is updated fine
5. Reapply the changes again, the manifest is still same from step 4

from webpack logs, it seems manifest is emitted fine but not written to disk

### Webpack version
4.4.1

### Webpack config

```js
  new WebpackAssetsManifest({
      writeToDisk: true,
      publicPath: true
    })
```

### Operating system
OSX

**Comment by webdeveric** [comment]
@gauravtiwari I have a fix in my `dev` branch. Can you try it out and let me know if it solves your issue?

**Comment by gauravtiwari** [comment]
@webdeveric Works great 👍 thanks for quick fix.

**Comment by gauravtiwari** [comment]
Perhaps make a minor release with this patch please?

**Comment by webdeveric** [comment]
@gauravtiwari I've published `v3.0.1`.

**Comment by gauravtiwari** [comment]
awesome, thank you so much ❤️
